docker-compose.yml

```diff
@@ -28,6 +28,7 @@ services:
       dockerfile: Dockerfile
     volumes:
         - ./src/django:/usr/local/src
+        - ./dist:/usr/local/src/static
```

Comment:
It is unclear why this volume mount is being added. I was able to comment it out and see changes to the React UI reflected on http://localhost:6543/ because in development the React UI  is managed by `react-scripts start`.

I think that this volume mount is making it seem like the steps added to `cibuild` are working, when the static asset bundle is never getting included in the Django container image.

Reply:
You're right. We only care about the static asset bundle being baked into container images. In development, we are using `$ react-scripts start`.